src/main/scala/org/constellation/snapshotstreaming/App.scala

```diff
@@ -12,19 +11,17 @@ import org.http4s.ember.client.EmberClientBuilder
 import org.typelevel.log4cats.slf4j.Slf4jLogger
 
 object App extends IOApp {
-  def mkLogger = Slf4jLogger.create[IO]
+  private val logger = Slf4jLogger.getLogger[IO]
```

Comment:
the previous implementation with `Slf4jLogger.create[IO]` was also correct, but then `mkLogger` was unused (could be just removed). anyway, both ways are correct imo